README.md

```diff
@@ -69,6 +69,40 @@ async fn main() {
 }
 ```
 
+Parameterized SQL Query
+
+```rust
+use spiceai::ClientBuilder;
+
+#[tokio::main]
+async fn main() {
+  let client = ClientBuilder::new()
+    .api_key("API_KEY")
+    .use_spiceai_cloud()
+    .build()
+    .await
+    .unwrap();
+
+  // Create a RecordBatch representing the values the parameters will bind https://docs.rs/arrow-flight/latest/arrow_flight/sql/client/struct.PreparedStatement.html#method.set_parameters
+  let fields = vec![
+    Arc::new(Field::new("$1", Int32, true)),
+    Arc::new(Field::new("$2", Float64, true)),
+  ];
```

Comment:
This is an extremely poor user experience to require the user to construct the parameter record batch manually. I don't think we should document this until we have a better way to use it.